app/models/etmpToUserAnswers/psaSchemeDetails/DirectorsOrPartnersTransformer.scala

```diff
@@ -24,24 +24,24 @@ import play.api.libs.json._
 
 class DirectorsOrPartnersTransformer @Inject()(addressTransformer: AddressTransformer) extends JsonTransformer {
 
-  def userAnswersDirectorReads(desPath: JsPath): Reads[JsObject] = {
-    userAnswersIndividualDetailsReads("directorDetails", desPath) and
-      userAnswersNinoReads("directorNino", desPath) and
-      userAnswersUtrReads(desPath) and
-      addressTransformer.getDifferentAddress(__ \ 'directorAddressId, desPath \ 'correspondenceAddressDetails) and
-      addressTransformer.getAddressYears(desPath, __ \ 'companyDirectorAddressYears) and
-      addressTransformer.getPreviousAddress(desPath, __ \ 'previousAddress) and
-      userAnswersContactDetailsReads("directorContactDetails", desPath) and
+  def userAnswersDirectorReads(ifPath: JsPath): Reads[JsObject] = {
```

Comment:
This parameter was introduced because we could have 2 possible paths here. Now since it is always going to be the same, we can hardcode the path and remove the parameter, just like it was before the toggle was added